test/gentlemanjerry.bats

```diff
@@ -149,3 +178,28 @@ teardown() {
   run timeout 10 java -cp /tmp/test SslTest api.logentries.com 25414
   [ "$status" -eq 0 ]
 }
+
+@test "Gentleman Jerry can connect to a TLSv1.2-only Endpoint" {
+  wait_for_tls12_server
+  run timeout 5 openssl s_client -CAfile /tmp/certs/jerry.crt -connect localhost:4433
+  [[ "$output" =~ "Verify return code: 0 (ok)" ]]
+
+  # Import cert into test truststore
+  keytool -importcert -file /tmp/certs/jerry.crt -keystore /tmp/certs/jerry.jks -storepass changeit -noprompt
+
+  # Set up JRuby and its gems
+  export PATH="/logstash-$LOGSTASH_VERSION/vendor/jruby/bin:$PATH"
+  export GEM_PATH="/logstash-$LOGSTASH_VERSION/vendor/bundle/jruby/1.9"
+
+  run jruby /tmp/test/manticore_test.rb https://localhost:4433 /tmp/certs/jerry.jks
```

Comment:
I think doing with the Ruby test is fine, but note that you could also use a Java implementation of this spec, which might be a little simpler. See `test/run_tests.sh` and `test/SslTest.java` if you're interested!

Reply:
Thanks! I implemented the Manticore-specific test early on when chasing a red herring that seemed to indicate that Manticore (and the underlying `org.apache.http.conn.ssl.SSLConnectionSocketFactory`) was behaving differently than SslTest.java (and `javax.net.ssl.SSLSocketFactory`), but this appears, thankfully, not to be the case.

I've added a commit which removes the Manticore test (and removes the test depending on the https://tlsv12-elb.aptible-test-grumpycat.com Endpoint, which is not guaranteed to exist).